server/api/v2/openapi.json

```diff
@@ -147,7 +147,7 @@
         ],
         "responses": {
           "200": {
-            "description": "Latest available report for the given cluster IDs. Returns rules and their descriptions that were hit by the cluster.",
+            "description": "Latest available report for the given organization and cluster combination. Returns rules and their descriptions that were hit by the cluster. Disabled rules are ommitted by default unless the corresponding query param is provided.",
```

Comment:
maybe instead of `corresponding query param` we could use `get_disabled param` to be more explicit